app/forms/waste_carriers_engine/payment_summary_form.rb

```diff
@@ -52,5 +51,18 @@ def assign_params(params)
     def paying_by_card?
       temp_payment_method == "card"
     end
+
```

Comment:
Thing something snuck in when resolving the conflicts or from previous. You can ditch everything from here downwards.